tests/utils/esx/govc.go

```diff
@@ -85,12 +85,17 @@ func ShutDownVM(vmName string) string {
 	return out
 }
 
-// GetDatastoreList returns a list of datastore names available
+// GetDatastoreList returns a list of datastore names available.
 func GetDatastoreList() []string {
 	// log.Printf("Finding Datastores available on ESX")
 	cmd := esx.DatastoreInfo + esxcliJSON + esx.DatastoreList
 	out, _ := ssh.InvokeCommandLocally(cmd)
-	return strings.Fields(out)
+	val := strings.SplitAfter(out, "\n")
```

Comment:
Similar comment to use Split instead of SplitAfter. With using split, you can also get rid of line 95